infra/modules/providers/azure/ad-application/output.tf

```diff
@@ -0,0 +1,14 @@
+output "application_id" {
+  description = "The client id output by the app registration."
+  value       = azuread_application.auth.*.application_id
+}
+
+output "azuread_config_data" {
+  description = "Output data that pairs azuread names with their client ids."
+  value = {
+    for azuread in data.azuread_application.auth :
+    azuread.name => {
+      client_id = azuread.application_id
```

Comment:
Would recommend leaving at application_id (instead of client_id =, use application_id =) since the client_id is from the perspective of the consumer of this config data.

Reply:
client_id was replaced with application_id in all places.